packages/jest-reporters/src/CoverageReporter.ts

```diff
@@ -148,7 +152,7 @@ export default class CoverageReporter extends BaseReporter {
         forkOptions: {serialization: 'json'},
         maxRetries: 2,
         numWorkers: this._globalConfig.maxWorkers,
-      });
+      }) as JestWorkerFarm<CoverageWorker>;
```

Comment:
Type casting is needed for now. Will work without casting after implementing `createWorkerFarm()`.